server/the-usher-openapi-spec.yaml

```diff
@@ -819,7 +819,13 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: "#/components/schemas/RoleWithPermissions"
+                  allOf:
+                    - $ref: "#/components/schemas/RoleWithPermissions"
+                    - type: object
+                      properties:
+                        assigned_at:
+                          type: string
```

Comment:
@k1ch  do you want to add a `description` here to make it clear what this timestamp field is representing?